jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/jaxrs31/ee/multipart/MultipartSupportIT.java

```diff
@@ -0,0 +1,426 @@
+                            .request(MediaType.MULTIPART_FORM_DATA_TYPE)
+                            .post(Entity.entity(new GenericEntity<>(multipart) {
+                            }, MediaType.MULTIPART_FORM_DATA))) {
+                Assertions.assertEquals(Response.Status.OK, response.getStatusInfo());
+                final List<EntityPart> entityParts = response.readEntity(new GenericType<>() {
+                });
+                if (entityParts.size() != 3) {
+                    final String msg = "Expected 3 entries, received " +
+                            entityParts.size() +
+                            '.' +
+                            System.lineSeparator() +
+                            getMessage(entityParts);
+                    Assertions.fail(msg);
+                }
+                EntityPart part = find(entityParts, "received-string");
+                Assertions.assertNotNull(part, getMessage(entityParts));
+                Assertions.assertEquals("test string", part.getContent(String.class));
+
+                part = find(entityParts, "received-file");
+                Assertions.assertNotNull(part, getMessage(entityParts));
+                Assertions.assertTrue(part.getContent(String.class).contains("value6"));
+
+                part = find(entityParts, "added-input-stream");
+                Assertions.assertNotNull(part, getMessage(entityParts));
+                Assertions.assertEquals("Add part on return.", part.getContent(String.class));
+
+                // Check headers.  Should be 4:  Content-Disposition, Content-Type, and the 2 headers
+                // that were added.
+                if ((part.getHeaders() == null) || (part.getHeaders().size() != 4)) {
+                    final String msg = "Expected 4 headers, received " +
+                            part.getHeaders().size();
+                    Assertions.fail(msg);
+                }
+                Assertions.assertEquals("Test1", part.getHeaders().get("Header1").get(0));
+                Assertions.assertEquals("Test2", part.getHeaders().get("Header2").get(0));
+            }
+        }
+    }
+
+    /**
+     * Verify that invoking a resource method with a single EntityPart can be injected in different formats; 
+     * {@link String}, {@link EntityPart} and {@link InputStream} with the same content for each.
+     *
+     * @throws Exception if an error occurs in the test
+     */
+    @Test
+    public void singleFormParamTest() throws Exception {
+        try (Client client = ClientBuilder.newClient()) {
+            final List<EntityPart> multipart = List.of(
+                    EntityPart.withName("test-string")
+                            .content("test single part")
+                            .mediaType(MediaType.TEXT_PLAIN_TYPE)
+                            .build());
+            try (
+                    Response response = client.target(createCombinedUri(uri, "test//single-form-param"))
```

Comment:
I think `test//single-form-param` should be `test/single-form-param`.

Reply:
+1 to this, nice catch!

Reply:
thanks